doc/api/process.markdown

```diff
@@ -114,6 +111,11 @@ is in an undefined state. Blindly resuming means *anything* could happen.
 Think of resuming as pulling the power cord when you are upgrading your system.
 Nine out of ten times nothing happens - but the 10th time, your system is bust.
 
+It is mainly useful to perform synchronous cleanup when the server is at a 
+problematic state in order to gracefully shut down.
+
+If you do use it, restart your application after every unhandled exception!
```

Comment:
this sounds right but I might say: 

Handling this type of exception recognizes an unrecoverable problem with the application, at which point it is time to both: *restart the application* and *investigate further*.